components/charts/Tooltip/index.js

```diff
@@ -0,0 +1,57 @@
+import { useRef, createElement } from "react";
+import PropTypes from "prop-types";
+import * as Styled from "./styles";
+
+const Tooltip = ({
+  value,
+  tooltipFormatter,
+  x,
+  y,
+  height = 16,
+  isVisible = false,
+}) => {
+  const label = tooltipFormatter ? tooltipFormatter(value) : value;
+  const width = label.length * 8;
+  const arrowWidth = 6;
+  const xCenter = x + width / 2;
+
+  return (
+    <Styled.Tooltip
+      transform={`translate(-${width / 2} -${height * 1.5})`}
+      opacity={isVisible ? 1 : 0}
```

Comment:
use opacity one the tooltip so that it is always in the accessibility tree and serves as the label for the bar data.